src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1539,8 +1589,16 @@ protected override void InvokeDefaultAction(string path)
                 if (isDirectory)
                 {
                     DirectoryInfo directory = new DirectoryInfo(path);
+                    InodeTracker tracker = null;
+
+                    GetChildDynamicParameters fspDynamicParam = DynamicParameters as GetChildDynamicParameters;
```

Comment:
I thought I had a check for recurse at one time. It must have gotten lost in the shuffle. Fixed.